tests/osgi-tests/src/test/java/org/wso2/extension/siddhi/io/http/test/osgi/HttpSink.java

```diff
@@ -0,0 +1,167 @@
+    private static final String CLIENTTRUSTSTORE_FILENAME = "client-truststore.jks";
+    private static final String KEYSTORESTORE_FILENAME = "wso2carbon.jks";
+
+    @Inject
+    private CarbonServerInfo carbonServerInfo;
+
+    @Inject
+    protected BundleContext bundleContext;
+
+    /**
+     * Replace the existing deployment.yaml file with populated deployment.yaml file.
+     */
+    private Option copyCarbonYAMLOption() {
+        Path carbonYmlFilePath;
+        String basedir = System.getProperty("basedir");
+        if (basedir == null) {
+            basedir = Paths.get(".").toString();
+        }
+        carbonYmlFilePath = Paths.get(basedir, "src", "test", "resources", "netty", DEPLOYMENT_FILENAME);
+        return copyFile(carbonYmlFilePath, Paths.get("conf", DEPLOYMENT_FILENAME));
+    }
+
+    /**
+     * Place default client  trusts-sore.jks file in conf security folder.
+     */
+    private Option copyCarbonClientTrustStoreOption() {
+        Path carbonYmlFilePath;
+        String basedir = System.getProperty("basedir");
+        if (basedir == null) {
+            basedir = Paths.get(".").toString();
+        }
+        carbonYmlFilePath = Paths.get(basedir, "src", "test", "resources", "security",
+                CLIENTTRUSTSTORE_FILENAME);
+        return copyFile(carbonYmlFilePath, Paths.get("resources", "security", CLIENTTRUSTSTORE_FILENAME));
+    }
+
+    /**
+     * Place default client  key-store.jks file in conf security folder.
+     */
+    private Option copyCarbonKeyStoreOption() {
+        Path carbonYmlFilePath;
+        String basedir = System.getProperty("basedir");
+        if (basedir == null) {
+            basedir = Paths.get(".").toString();
+        }
+        carbonYmlFilePath = Paths.get(basedir, "src", "test", "resources", "security",
+                KEYSTORESTORE_FILENAME);
+        return copyFile(carbonYmlFilePath, Paths.get("resources", "security", KEYSTORESTORE_FILENAME));
+    }
+
+    @Configuration
+    public Option[] createConfiguration() {
+        return new Option[]{copyCarbonYAMLOption(),
+                copyCarbonClientTrustStoreOption(),
+                copyCarbonKeyStoreOption(),
+                CarbonDistributionOption.carbonDistribution(maven()
+                        .groupId("org.wso2.extension.siddhi.io.http")
+                        .artifactId("org.wso2.extension.io.http.test.distribution")
+                        .type("zip")
+                        .versionAsInProject()),
+                copyOSGiLibBundle(maven()
+                        .artifactId("siddhi-io-http")
+                        .groupId("org.wso2.extension.siddhi.io.http")
+                        .versionAsInProject()),
+                systemProperty("java.security.auth.login.config")
+                        .value(Paths.get("conf", "security", "carbon-jaas.config").toString())
+                //CarbonDistributionOption.debug(5005)
```

Comment:
Remove commented code

Reply:
Fixed in 156b93ab839dfc225e0a915f8951b61db7afa2ab